pkg/apis/hawtio/v1/hawtio_types.go

```diff
@@ -72,6 +72,10 @@ type HawtioSpec struct {
 	Route HawtioRoute `json:"route,omitempty"`
 	// List of external route names that will be annotated by the operator to access the console using the routes
 	ExternalRoutes []string `json:"externalRoutes,omitempty"`
+	// The Hawtio console container image version.
+	// Deprecated and only remains for legacy purpose in respect of older operators (<1.0.0)
```

Comment:
A correct godoc deprecation notation should be used. The tool should then treat it accordingly.
https://stackoverflow.com/questions/7849663/how-do-you-mark-code-as-deprecated-in-go